### PR TITLE
dev-python/nautilus-python: rename USE doc -> gtk-doc

### PR DESCRIPTION
**Pull request by sevz17**
install examples anyway

Signed-off-by: Leonardo Hernández Hernández <leohdz172@protonmail.com>

### CODE REVIEW

File: dev-python/nautilus-python/nautilus-python-4.0.ebuild [hunk 1]
```diff
@@ -28,21 +28,22 @@ RDEPEND="
...
-		$(meson_feature doc docs)
+		$(meson_feature gtk-doc docs)
 	)
 	meson_src_configure
 }
 
 src_install() {
 	meson_src_install
 
-	use doc && mv "${ED}/usr/share/doc/${PN}/"* "${ED}/usr/share/doc/${P}" || die
```
Comment (sevz17) [inline dev-python/nautilus-python/nautilus-python-4.0.ebuild:45]: When I tried to install with `-doc` it just called die

**Comment by mattst88** [comment]
> install examples anyway

Do you care that the examples are installed? Should we maybe just remove them instead?

File: dev-python/nautilus-python/nautilus-python-4.0.ebuild (hunk 1 above)
Comment (mattst88) [reply dev-python/nautilus-python/nautilus-python-4.0.ebuild:45]: Ah, probably because you don't have gtk-doc installed and I didn't add the dependency?

**Comment by sevz17** [comment]
> Do you care that the examples are installed? Should we maybe just remove them instead?

They are 7 files, each one less than 1mb (compressed), if you want I can add `examples`

File: dev-python/nautilus-python/nautilus-python-4.0.ebuild (hunk 1 above)
Comment (sevz17) [reply dev-python/nautilus-python/nautilus-python-4.0.ebuild:45]: I don't think so:

```sh
false && echo "done" || echo "not done"
true && echo "done" || echo "not done"
```

the first one will print "not done" (equivalent to `die` here)

**Comment by mattst88** [comment]
I guess I'll take that as a "yes".

I don't have a strong preference. I just haven't found installed examples to ever really be useful.

File: dev-python/nautilus-python/nautilus-python-4.0.ebuild (hunk 1 above)
Comment (mattst88) [reply dev-python/nautilus-python/nautilus-python-4.0.ebuild:45]: Ah, thanks.

**Pull request closed without merging**